pkg/util/iptables/iptables_test.go

```diff
@@ -1198,3 +1198,152 @@ func TestRestoreAllWaitBackportedIptablesRestore(t *testing.T) {
 		t.Errorf("expected failure")
 	}
 }
+
+// TestExtractLines tests that
+func TestExtractLines(t *testing.T) {
+	type args struct {
```

Comment:
This extra struct level doesn't make it more readable.  I'd just inline these 3 fields to the test struct